runtime/src/main/java/org/corfudb/runtime/exceptions/TrimmedException.java

```diff
@@ -1,15 +1,27 @@
 package org.corfudb.runtime.exceptions;
 
+import lombok.Getter;
+import lombok.Setter;
+
 /**
  * This exception is thrown when a client tries to read an address
  * that has been trimmed.
  */
 public class TrimmedException extends LogUnitException {
-    public TrimmedException() {
 
+    /*
+     * This flag determins whether the operation that caused this exception
+     * can retry or not.
+     */
+    @Getter
+    @Setter
+    private boolean retriable = true;
```

Comment:
No, I still disagree. What is a retriable exception? The exception is not retriable, the read of the object at the version is, and that is precisely because whether or not the address is trimmed is unresolved.